src/components/radio-group/radio/bl-radio.test.ts

```diff
@@ -19,7 +19,7 @@ describe("bl-radio", () => {
       `<div
         aria-disabled="false"
         aria-labelledby="label"
-        aria-readonly="false"
```

Comment:
I don’t quite understand why you did this

Reply:
aria-readonly property is not supported in Radio Group according to [W3C ARIA Radio Group Example](https://www.w3.org/WAI/ARIA/apg/patterns/radio/examples/radio-rating/)